os/sdk-modifying-coreos.md

```diff
@@ -61,57 +61,6 @@ cork enter
 
 To use the SDK chroot in the future, run `cork enter` from the above directory.
 
-### Option 2: using repo and cros\_sdk
-
-#### Install Repo
-
-The `repo` utility helps to manage the collection of git repositories that makes up Container Linux. 
-
-For newer Debian, Ubuntu, and other Debian based systems, install the repo package from your distro:
-
-    sudo apt-get install repo
-
-For systems without a packaged repo download it and add it to `$PATH`:
-
-```sh
-mkdir ~/bin
-export PATH="$PATH:$HOME/bin"
-curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
-chmod a+x ~/bin/repo
-```
-
-You may want to add the `PATH` export to `.bashrc` or `/etc/profile.d/` so that you don’t need to set `$PATH` in every new shell.
-
-#### Bootstrap the SDK chroot
-
-Create a project directory. This will hold all of your git repos and the SDK chroot. A few gigabytes of space will be necessary.
-
-```sh
-mkdir coreos; cd coreos
-```
-
-Initialize the .repo directory with the manifest that describes all of the git repos required to get started.
-
-```sh
-repo init -u https://github.com/coreos/manifest.git
-```
-
-Synchronize all of the required git repos from the manifest.
-
-```sh
-repo sync
-```
-
-#### Use cros\_sdk to setup the chroot
-
-Download and enter the SDK chroot which contains all of the compilers and tooling.
-
-```sh
-./chromite/bin/cros_sdk
-```
-
-**WARNING:** To delete the SDK chroot, use `./chromite/bin/cros_sdk --delete`. Otherwise, you will delete `/dev` entries that are `bind`-mounted into the chroot.
```

Comment:
I'm not sure, but cork does have `cork delete` which will delete the sdk chroot for you, so I'll include that information in the same style.